src/lib/pages/backoffice/Item/ItemForms/ItemEditor/ItemForm/ItemForm.js

```diff
@@ -99,8 +99,12 @@ export class ItemForm extends Component {
       successSubmitMessage,
       title,
       pid,
+      itemPastLoans,
     } = this.props;
     const initialValues = data ? this.prepareData(metadata) : {};
+    const hasPastLoans = itemPastLoans > 0;
+    const message =
+      'Not possible to edit the document field if the item already has past loans.';
```

Comment:
Very very very minor:
```suggestion
      'Change of document is disabled because this item has active or past loans.';
```